scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -335,7 +335,7 @@
     (let [timer (timers/start (metrics/timer "cc-pending-offers-compute" name))
           pods (add-starting-pods this @all-pods-atom)
           nodes @current-nodes-atom
-          offers-this-pool (generate-offers this (@pool->node-name->node pool-name)
+          offers-this-pool (generate-offers this (get @pool->node-name->node pool-name {})
```

Comment:
Would it make sense to extract a function for this since we're doing it in two places?

Reply:
I can do that.